packages/core/stories/patterns/vertical-navigation/vertical-navigation.stories.tsx

```diff
@@ -0,0 +1,247 @@
+          children: [{ name: "Monitoring", href: "#" }],
+        },
+      ],
+    },
+    { name: "Trades", href: "#" },
+    { name: "Reports", href: "#" },
+  ];
+
+  const [active, setActive] = useState(navigationData[0].name);
+  const [expanded, setExpanded] = useState<string[]>([]);
+
+  const handleExpand = (item: NavigationItemData) => () => {
+    const isExpanded = expanded.includes(item.name);
+    setExpanded(
+      isExpanded
+        ? expanded.filter((name) => name !== item.name)
+        : [...expanded, item.name]
+    );
+  };
+
+  const isParentOfActiveItem = (
+    children: NavigationItemData[],
+    activeName: string
+  ): boolean => {
+    return children.some((child: NavigationItemData) => {
+      if (child.name === activeName) return true;
+      return child.children
+        ? isParentOfActiveItem(child.children, activeName)
+        : false;
+    });
+  };
+
+  const RecursiveNavItem: FC<{ item: NavigationItemData }> = ({ item }) => {
+    return (
+      <li style={{ listStyle: "none" }} key={item.name}>
+        <NavigationItem
+          active={active === item.name}
+          blurActive={
+            item.children && !expanded.includes(item.name)
+              ? isParentOfActiveItem(item.children, active)
+              : false
+          }
+          href={item.href}
+          expanded={expanded.includes(item.name)}
+          orientation="vertical"
+          onClick={(event) => {
+            // prevent default to avoid navigation in storybook example
+            event.preventDefault();
+            setActive(item.name);
+          }}
+          level={item.level || 0}
+          onExpand={handleExpand(item)}
+          parent={!!item.children}
+        >
+          {item.name}
+        </NavigationItem>
+        {item.children &&
+          expanded.includes(item.name) &&
+          item.children.map((child) => (
+            <RecursiveNavItem
+              item={{ ...child, level: (item.level || 0) + 1 }}
+            />
+          ))}
+      </li>
+    );
+  };
+
+  return (
+    <BorderLayout>
+      <BorderItem position="north">
+        <Header />
+      </BorderItem>
+      <BorderItem
+        position="west"
+        style={{
+          marginTop: "calc(var(--salt-spacing-300) * 2)", // Margin should be the height of the header
+          position: "fixed",
+        }}
+      >
+        <aside style={{ width: "250px" }}>
+          <nav>
+            <StackLayout
+              gap="var(--salt-size-border)"
+              as="ul"
+              style={{ listStyle: "none", margin: 0, padding: 0 }}
+            >
+              {navigationData.map((item) => (
+                <RecursiveNavItem item={item} />
+              ))}
+            </StackLayout>
+          </nav>
+        </aside>
+      </BorderItem>
+      <BorderItem
+        position="center"
+        style={{
+          marginTop: "calc(var(--salt-spacing-300) * 2)",
+          marginLeft: "250px",
```

Comment:
Yeah I didn't know how to explicitly ask this but was hoping the PR would allow everyone to see if it's what we want.

Did you find a better way when making the Salt demo apps? Did you use border layout?

Reply:
No, i wasn't use border layout. 

Can this be removed if fixed position is applied not on BorderItem level?

Reply:
I've pushed an alternative way using sticky prop, there's slightly less configuration for the user.